pkg/shell/hosts.jsx

```diff
@@ -96,10 +96,8 @@ export class CockpitHosts extends React.Component {
     }
 
     toggleMenu() {
-        if (this.state.opened)
-            document.getElementById(this.props.selector).className = "area-ct-subnav sidebar";
-        else
-            document.getElementById(this.props.selector).className = "area-ct-subnav sidebar interact";
+        document.getElementById(this.props.selector).classList.toggle("interact", !this.state.opened);
```

Comment:
FYI: [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) is much better... specifically `.classList.toggle()` (or `.add()` and `.remove()`, depending) is _much preferred_ to modifying the `className` string, as it's native API to add/remove/toggle classes instead of mucking about with strings. We _also_ don't have to do an if / else block here as a result. :wink:

The reason I changed it: I was running into the issue where I added a class on the element in the previous code was removing it, as the old code was sledgehammering the specific classes instead of directly modifying which ones are on the element.